pytorch_blade/bazel_build.py

```diff
@@ -66,6 +66,10 @@ def __init__(self, *args, **kwargs):
             "--action_env TORCH_BLADE_TORCH_INSTALL_PATH={}".format(self.torch_dir),
             # Workaroud issue: https://github.com/bazelbuild/bazel/issues/10327
             "--action_env BAZEL_LINKLIBS=-lstdc++",
+            "--action_env CC={}".format(which("gcc")),
+            "--action_env CXX={}".format(which("g++")),
+            # for onednn cmake external build
+            "--action_env IF_CXX11_ABI={}".format(int(self.GLIBCXX_USE_CXX11_ABI)),
```

Comment:
Nit: I prefer to add a config such as `--config=torch_cxx11abi` that will set both config of `cxx11abi_1` and `--action_env IF_CXX11_ABI=1` in `.bazelrc`

Reply:
done